.changeset/dark-pears-argue.md

```diff
@@ -0,0 +1,12 @@
+---
+"@ijlee2-frontend-configs/eslint-config-ember": minor
+"@ijlee2-frontend-configs/eslint-config-node": minor
```

Comment:
In case the breaking changes from [`eslint-import-resolver-typescript@v4`](https://github.com/import-js/eslint-import-resolver-typescript/releases/tag/v4.0.0) could affect end-projects.